CSS_Tasks/CSS_Advanced/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <title>WebGL</title>
+    <link href="stylesheet.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <button type="submit" class="button buttonGreenStyle"> Вход </button>
+        <button type="submit" class="button buttonGreenStyle"> Регистрация </button>
+    </header>
+
+    <main class="pageContainer">
+        <div class="navigationBar">
+
+            <nav>
+                <ul>
+                    <li>Компютърна графика с WebGL </li>
+                    <hr>
+                    <li>Програмиране с Gо</li>
+                    <hr>
+                    <li>Програмиране с Ruby</li>
+                    <hr>
+                    <li>Agile</li>
+                    <hr>
+                </ul>
+            </nav>
+
+        </div>
+        <div class="content">
+
+            <h1> Компютърна графика с WebGL</h1>
+            <hr>
+
+            <h2>доц. П. Бойчев</h2>
+            <p>След завършване на курса студентите ще могат да създават тримерни статични изображения и тримерни анимации</p>
+
+            <p><i>Тип: </i> задължителна избираема</p>
+            <p><i>Курс: </i> 1 <sup>ви</sup></p>
+            <p><i>Програма: </i> бакалавър</p>
+
+            <textarea placeholder="Добавяне на коментар"></textarea>
+            <div class="mainButtons">
+                <button type="submit" class="button buttonWhiteStyle"> Отказ </button>
```

Comment:
You may want to change the mouse pointer style to a hand when you hover over the buttons. And also when clicking on the buttons there is this black border that you also may want to remove.

Reply:
Same goes for the elements in the navigation pane. No need to be real urls, just changing the pointer style.

Reply:
Same goes for the buttons in the top right corner of the screen.

Reply:
And also for the names of the people that have commented - they are in dark blue, that indicates usually urls, which can be clicked (so the mouse pointer style should be different too).